base/replutil.jl

```diff
@@ -414,8 +414,14 @@ function showerror_ambiguous(io::IO, meth, f, args)
         i < length(p) && print(io, ", ")
     end
     print(io, ") is ambiguous. Candidates:")
+    sigfix = Any
     for m in meth
         print(io, "\n  ", m)
+        sigfix = typeintersect(m.sig, sigfix)
+    end
+    if isa(unwrap_unionall(sigfix), DataType) && sigfix <: Tuple
+        print(io, "\nPossible fix: ")
```

Comment:
Right. The idea here is re-introduce an analogous show for that last piece of information. Right now, we have the initial bit of information, rearranged somewhat because now we have an example of a failing method call and the methods themselves have no ordering:
```julia
ERROR: MethodError: f(::Int64, ::Int64) is ambiguous. Candidates:
  f(::Any, ::Integer) in Main at REPL[2]:1
  f(:: Integer, ::Any) in Main at REPL[1]:1
```

This PR would add showing of one possible way of fixing the method definition. Unlike before, the solution was not unique, nor does it need to be defined before, so the rest of the old text doesn't apply. This proposes one example method that would solve the ambiguity between the printed methods for solving this MethodError. However, it doesn't print all of the signatures that are needed to "fix" the ambiguity (that's what the ambiguity test is for instead).

So for the above example, this PR would additionally print:
```julia
Possible fix: f(::Integer, ::Integer)
```
Note that the set of "to fix" options is much larger. For instance, using the exact signature printing on the first line is also a valid fix.